src/tools/cgset.c

```diff
@@ -293,6 +417,25 @@ STATIC int parse_r_flag(const char * const program_name, const char * const name
 }
 
 #ifndef UNIT_TEST
+static int add_subtree_control_name_value(struct control_value *name_value)
+{
+	if (cgrp_subtree_ctrl_val) {
+		err("%s: duplicate -r %s option found\n", program_name, name_value->name);
+		return -1;
```

Comment:
It feels weird to return -1 on error here.  Would it be better to use `ECG<WHATEVER>` instead?

Reply:
ah ! will change to `ECGFAIL`